beta/src/pages/learn/updating-objects-in-state.md

```diff
@@ -17,7 +17,7 @@ State can hold any kind of JavaScript value, including objects. But you shouldn'
 
 </YouWillLearn>
 
-## What's a mutation? {/*whats-a-mutation*/}
+## What's a mutation? {/_whats-a-mutation_/}
```

Comment:
How did these heading id's change? We do not generate ids on pre commit anymore 🤔. This is affecting multiple files. Can we restore it to older syntax of IDs ? 
```diff
+{/*whats-a-mutation*/}
-{/_whats-a-mutation_/}
```